package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cherrytwist-server",
-  "version": "0.1.3",
+  "version": "0.1.5",
```

Comment:
npm-version should be used here, now this PR stood for a while and could have lead to conflicts with the manually bumped version

Reply:
npm version handling is something we need to sort properly - for now just had to get it up so that it can be identified.